adoc/extensions/sycl_khr_max_work_group_query.adoc

```diff
@@ -0,0 +1,122 @@
+[[sec:khr-max-work-group-query]]
+= sycl_khr_max_work_group_id
+
+This extension allows developers to query iteration bounds in each dimension for a ND-range kernel.
+The application must ensure that the number of work-groups in each dimension [code]#N# of an ND-range kernel is less than or equal to [code]#max_work_group_id<N>#.
+
+[[sec:khr-max-work-group-query-dependencies]]
+== Dependencies
+
+This extension does not depend on other extensions.
+
+[[sec:khr-max-work-group-query-feature-test]]
+== Feature test macro
+An implementation supporting this extension must predefine the [code]#SYCL_KHR_MAX_WORK_GROUP_QUERY# macro to one of the values defined in the table below.
+
+[%header,cols="1,5"]
+|===
+|Value
+|Description
+
+|1
+|Initial version of this extension.
+|===
+
+== New device descriptors
+
+.[apidef]#khr::info::device::max_work_group_id#
+[source,role=synopsis,id=api:khr-info-device-max-work-group-query]
+----
+namespace sycl::khr::info::device {
+template<int Dimensions = 3>
+struct max_work_group_id {
+    using return_type = range<Dimensions>;
+};
+} // namespace sycl::khr::info::device
+----
+
+_Remarks:_ Template parameter to [api]#device::get_info#.
+
+_Constraints:_ Available only when [code]#Dimensions# is 1, 2 or 3.
+
+_Returns:_ The maximum number of work-groups that can be submitted in each dimension of an [code]#nd_range<Dimensions># kernel. The minimum value is 1 in each dimension if the device is not [code]#info::device_type::custom#.
```

Comment:
```suggestion
_Returns:_ The maximum number of work-groups that can be submitted in each dimension of an [code]#nd_range<Dimensions># kernel. The minimum value is 1 in each dimension if the device is not [api]#info::device_type::custom#.
```

Kind of an editorial nit, but this will make it a link to the definition.